flowetl/Pipfile

```diff
@@ -4,13 +4,15 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-apache-airflow = {extras = ["crypto"],version = "==2.2.3"}
```

Comment:
crypto extra is no longer required?

Reply:
It's included in the airflow image itself - various airflow requirements are put in here to expose them to the graph builder and test suite.